### PR TITLE
Seed users and items and comments

### LINKED ISSUE #2: Add seeds

### PR DESCRIPTION
**Pull request by alexkalderimis**
# Description

Fixes #2, seeing users and items

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix
- [x ] Tooling
- [ ] A documentation update
- [ ] Refactoring

# How Has This Been Tested?

Manually with `rake db:seed` and inspection at the console.

# Checklist:

- [x ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [ ] I added the relevant people as reviewers to my PR

### CODE REVIEW

**Comment by vanessa-cooper** [comment]
That was fast! Thanks for the PR @alexkalderimis, I'll review the changes right away.

File: backend/Gemfile [hunk 1]
```diff
@@ -4,6 +4,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby '2.7.0'
 
 gem 'dotenv-rails', groups: [:development, :test]
+gem 'pry-rails', groups: [:development]
```
Comment (alexkalderimis) [inline backend/Gemfile:7]: Added because irb is broken in some environments (broken `reline`)

File: backend/app/models/user.rb [hunk 2]
```diff
@@ -13,10 +13,11 @@ class User < ApplicationRecord
   acts_as_follower
   acts_as_followable
 
-  validates :username, uniqueness: { case_sensitive: true },
-                       format: { with: /\A[a-zA-Z0-9]+\z/ },
-                       presence: true,
-                       allow_blank: false
+  validates :username,
+    uniqueness: { case_sensitive: true, message: 'not available' },
+    format: { with: /\A[._a-zA-Z0-9]+\z/, message: ->(user, data) { "only allows letters and digits. Got #{data[:value]}" } },
```
Comment (alexkalderimis) [inline backend/app/models/user.rb:18]: Changed because the username format is unnecessarily restrictive and would only annoy users

**Review by vanessa-cooper — APPROVED** [review]
Nice! Looks like there are some records in your local DB. Now you can merge the PR and help us with the bug. Take a quick coffee break and [click here](https://app.wilco.gg/home) once you’re ready for your next project.

**Pull request merged**